WordleGame/WordleGame/RecordsViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  RecordsViewController.swift
+//  WordleGame
+//
+//  Created by Alex on 16.05.22.
+//
+
+import UIKit
+
+class RecordsViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    
+    let gameResults = UserDefaultsService.shared.getGameResults()
+    var gameResultsToTableView = [String]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        settingsBarItem()
+        
+        gameResultsToTableView = gameResults.compactMap({$0.userName + " / Score: " + String($0.score) + " / Time: " + String($0.time)})
```

Comment:
There's no need to modify your array here. It's better to do it in `tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell`